src/analyticsTracking/AmplitudeProxy.js

```diff
@@ -0,0 +1,78 @@
+import {Amplitude} from '@amplitude/react-amplitude';
+import {memoize} from '@amplitude/react-amplitude/src/lib/memoize';
+import debounce from 'lodash.debounce';
+import {buttonTracking} from './eventUtils';
+import {EVENT_CONSTANTS} from '../analyticsTracking';
+
+/**
+ *
+ */
+class AmplitudeProxy extends Amplitude {
+  logToConsole = false;
+
+  constructor(props) {
+    super(props);
+    if (typeof props.debounceInterval === 'number') {
+      this.logEvent = debounce(this.dispatch, props.debounceInterval);
```

Comment:
this swaps out the implementation with to call the new `dispatch` method instead of the `_makeLogEvent` method allowing us to _hook_ into events to call things like HotJars code or push the events into the sessionStorage for testing